src/frontend/http.rs

```diff
@@ -71,7 +71,7 @@ async fn delete_handler(req: Request<Body>) -> Result<Response<Body>> {
     let lsm = req.data::<Arc<RwLock<LSM>>>().unwrap();
     let mut lsm = lsm.write().unwrap();
 
-    lsm.put(key, Value(None))?;
+    lsm.put(key, Value(OptDatum::Tombstone))?;
```

Comment:
I thought deletions would be exposed via `delete(k)`?

Reply:
Yeah absolutely, let me do that. "Tombstone" should be a storage-internal concept.